code/modules/mob/living/carbon/human/species/shadow.dm

```diff
@@ -0,0 +1,25 @@
+/datum/species/shadow
+	name = "Shadow"
+
+	icobase = 'icons/mob/human_races/r_shadow.dmi'
+	deform = 'icons/mob/human_races/r_shadow.dmi'
+
+	default_language = "Galactic Common"
+	unarmed_type = /datum/unarmed_attack/claws
+	light_dam = 2
+	darksight = 8
+
+	blood_color = "#CCCCCC"
+	flesh_color = "#AAAAAA"
+
+	has_organ = list(
+		"brain" = /obj/item/organ/brain/slime
+		)
+
+	flags = NO_BLOOD | NO_BREATHE | NO_SCAN
+	bodyflags = FEET_NOSLIP
```

Comment:
Again, I don't think this was a thing before, correct me if I am wrong. Not sure why they would have noslip either.